packages/snap/src/features/confirmation/views/ConfirmTransactionRequest/render.test.tsx

```diff
@@ -26,6 +26,46 @@ import { ConfirmTransactionRequest } from './ConfirmTransactionRequest';
 import { DEFAULT_CONFIRMATION_CONTEXT } from './render';
 import type { ConfirmTransactionRequestContext } from './types';
 
+// Mock the transactionScanService
+jest.mock('../../../../snapContext', () => ({
+  ...jest.requireActual('../../../../snapContext'),
+  transactionScanService: {
+    scanTransaction: jest
+      .fn()
+      .mockResolvedValue(MOCK_SCAN_TRANSACTION_RESPONSE),
+  },
+}));
```

Comment:
we are able to mock network calls o at the edge using the mock server. Don't need to `jest.mock` it